pkg/torcx/paths.go

```diff
@@ -26,11 +26,14 @@ const (
 	DefaultBaseDir = "/var/lib/torcx/"
 	// DefaultConfDir is the default torcx config directory
 	DefaultConfDir = "/etc/torcx/"
+
 	// VendorStorePath is the vendor store path
-	VendorStorePath = VENDOR_DIR + "/store/"
+	VendorStorePath = VendorDir + "store/"
+	// VendorProfile is the vendor (lower) profile
+	VendorProfile = VendorDir + VendorProfileName + ".json"
```

Comment:
I'm not sure why we'd keep `/usr/share/torcx/vendor.json` if we have `/usr/share/torcx/profiles`, but I don't think there is a problem with it.